web-api/src/lambdaWrapper.js

```diff
@@ -26,21 +28,29 @@ exports.lambdaWrapper = lambda => {
       queryStringParameters: req.query,
     };
 
+    if (shouldMimicApiGateway) {
+      res.status(204).send('');
+    }
+
     const response = await lambda({
       ...event,
       body: JSON.stringify(req.body),
       logger: req.locals.logger,
     });
 
-    res.status(response.statusCode);
+    if (!shouldMimicApiGateway) {
+      res.status(response.statusCode);
 
-    res.set({
-      ...response.headers,
-      'X-Terminal-User': isTerminalUser,
-      ...exports.headerOverride,
-    });
+      res.set({
+        ...response.headers,
+        'X-Terminal-User': isTerminalUser,
+        ...exports.headerOverride,
+      });
+    }
 
-    if (
+    if (shouldMimicApiGateway) {
+      // Do nothing
+    } else if (
```

Comment:
Looks like the `else if` part of this `if` statement can be combined with the `if` statement above.